internal/provider/resource_environment.go

```diff
@@ -223,6 +223,34 @@ func resourceEnvironment() *schema.Resource {
 					},
 				},
 			},
+			"repositories": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
```

Comment:
Can we add descriptions to these fields? 
I think technically the ID here is of the repository/installation, right?

Reply:
I think so.

Reply:
sure

Reply:
I rescind my comment. 
I see there's an `repositories` object above.